qa/mixed-cluster/src/test/java/org/opensearch/backwards/IndexingIT.java

```diff
@@ -113,8 +113,6 @@ private void printClusterRouting() throws IOException, ParseException {
     /**
      * This test verifies that segment replication does not break when primary shards are on lower OS version. It does this
      * by verifying replica shards contains same number of documents as primary's.
-     *
```

Comment:
Changes in this file will be added as part of https://github.com/opensearch-project/OpenSearch/pull/8314. Reviewers can ignore these changes in this PR.